File: web3rpc/rpc-specs/code-samples/python/eth/miscellaneous/submitHashrate.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

hashrate = "0x5"
hashrateId = "0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.submit_hashrate(hashrate, hashrateId)

print(eth_response)

File: web3rpc/rpc-specs/code-samples/javascript/eth/miscellaneous/submitHashrate.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const hashrate = "0x5";
  const id =
    "0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef";

  provider.eth
    .submitHashrate(hashrate, id, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
